python/lsst/meas/algorithms/__init__.py

```diff
@@ -23,7 +23,26 @@
 """lsst.meas.algorithms
 """
 from __future__ import absolute_import
-from .algorithmsLib import *
+
+import lsst.afw.image
+import lsst.afw.math
+from lsst.afw.image import DefectBase
```

Comment:
I would use `py::module::import("lsst.afw.image")`.  I think we should import at sibling level only within the same package.